### PR TITLE
Add /disttocore

### PR DESCRIPTION
**Pull request by Salminar**
Disttocore v 1.1 final

### CODE REVIEW

**Review by nthirtyone — CHANGES_REQUESTED** [review]
With a command this simple it is hard to find anything wrong. But. It is missing `readme` entry.

Just add it (follow general pattern) and it should be good (if nobody notices anything else).

File: README.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ Gets the position of a player. Usage: `/whereis <name>`
 Lists possible variables for `/inventory` or `/crew`. Usage: 
 `/list <type>`
 `/list help` or `/list` for help.
-
+## /disttocore
+Display the distance in sectors between the player and the center of the galaxy. Usage `/disttocore`
```
Comment (mark-kubacki) [inline README.md:28]: Display**s**

(no blocker, though)

File: scripts/commands/disttocore.lua [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function execute(sender, commandName, ...)
+   	Player(sender):addScriptOnce("cmd/disttocore.lua")
+    return 0, "", ""
```
Comment (mark-kubacki) [inline scripts/commands/disttocore.lua:3]: Please use tabs only.

File: scripts/player/cmd/disttocore.lua [hunk 3]
```diff
@@ -0,0 +1,10 @@
+if onServer() then
+
+function initialize()
+	player = Player()
+	local distanceFromCenter = length(vec2(Sector():getCoordinates()))
+	player:sendChatMessage("Server", 0, "Distance from core : "..math.ceil(distanceFromCenter*10)*.1)
```
Comment (mark-kubacki) [inline scripts/player/cmd/disttocore.lua:6]: `"Distance from \\s(0:0): "…`

A community could refer to the area encompassed by the *barrier* as the *core*, so using `(0:0)` as reference removes any confusion. (The markup `\\s(…:…)` will turn the coordinates into a link that, if clicked on it, opens the galaxy map.)

**Comment by nthirtyone** [comment]
Merging, I will fix "issues" with upcoming changes.

**Pull request merged**